src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationOptions.cs

```diff
@@ -72,6 +73,8 @@ public string CookieName
 
         /// <summary>
         /// If set this will be used by the CookieAuthenticationMiddleware for data protection.
+        /// he default data protection service is based on machine key when running on ASP.NET, 
```

Comment:
You can just remove this comment, that's an implementation detail of DataProtection.